data/documentation.md

```diff
@@ -64,13 +64,13 @@ We only use numbered lists for describing steps as part of a process. Do this by
 
 `s3. Love numbers.`
 
-This looks like:
+This example follows the [style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#steps) and looks like:
 
 s1. Add numbers.
 s2. Check numbers.
 s3. Love numbers.
 
-Steps need an extra line break after the final step (ie 2 full blank lines). If you don't do this, other markdown directly after them won't work. If you have a subheading after numbered steps, add a line break after this.
+Steps need an extra line break after the final step (in other words, 2 full blank lines). If you don't do this, other markdown directly after them won't work. If you have a subheading after numbered steps, add a line break after this.

```

Comment:
Thanks Nick. I got this reviewed by another content designer before submitting the request, and we agreed it would be better to retain the 'in other words'.